c/tskit/tables.c

```diff
@@ -6592,44 +6662,42 @@ tsk_table_collection_print_state(tsk_table_collection_t *self, FILE *out)
 }
 
 int TSK_WARN_UNUSED
-tsk_table_collection_init(tsk_table_collection_t *self, tsk_flags_t options)
+tsk_table_collection_init(tsk_table_collection_t *self, tsk_flags_t TSK_UNUSED(options))
```

Comment:
Is the method signature kept the same for backwards compatibility? Or in case we need flags in future?

Reply:
Both maybe, but more the latter.